src/seaborn_image/_filters.py

```diff
@@ -41,6 +41,8 @@ def filterplot(
     interpolation=None,
     robust=False,
     perc=(2, 98),
+    diverging=False,
+    vmaxabs=None,
```

Comment:
Q: why do we need this argument? I think what you are proposing can be achieved with only `vmin` and `vmax`. 

If a user specifies `vmax` and `vmin`, we explicitly set it to those values (even when `diverging=True`)

If a user does not specify either `vmin` or `vmax`, we can calculate the `vmaxabs` ourselves. I don't see the need for a new argument.

Reply:
You are right. I added the new argument to not have to deal with checking if vmin and vmax are correct, but now that i think about it I'm doing the same checks but on a new variable. I will be changing this, thank you.